tardis/model/tests/test_base.py

```diff
@@ -413,3 +415,34 @@ def test_hdf_model_nparray(hdf_file_path, simulation_verysimple, attr):
     if hasattr(actual, "cgs"):
         actual = actual.cgs.value
     assert_almost_equal(actual, expected.values)
+
+
+@pytest.fixture(scope="module")
+def composition_from_config(atomic_data_fname):
+    filename = "tardis_configv1_verysimple.yml"
+    config = Configuration.from_yaml(data_path(filename))
+    config["atom_data"] = atomic_data_fname
+
+    simulation = Simulation.from_config(config)
+    composition = Composition(
+        simulation.model.density,
+        simulation.model.abundance,
+        simulation.plasma.atomic_mass,
+    )
+    return composition
+
+
+def test_composition_number_density(composition_from_config):
```

Comment:
The expected values here should be in reference data rather than calculated in the test